samples/bluetooth/peripheral_nfc_pairing/prj.conf

```diff
@@ -15,11 +15,19 @@ CONFIG_NFC_NDEF_RECORD=y
 CONFIG_NFC_NDEF_LE_OOB_REC=y
 CONFIG_NFC_NDEF_CH_MSG=y
 
+CONFIG_NFC_NDEF_TNEP_RECORD=y
+CONFIG_NFC_TNEP_TAG=y
+CONFIG_NFC_NDEF_PARSER=y
+CONFIG_NFC_NDEF_CH_PARSER=y
+CONFIG_NFC_NDEF_LE_OOB_REC_PARSER=y
+CONFIG_NFC_TNEP_CH=y
+
 CONFIG_BT=y
 CONFIG_BT_SMP=y
 CONFIG_BT_SMP_APP_PAIRING_ACCEPT=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_DEVICE_NAME="Nordic_NFC_pairing"
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
I'm curios why this was added, is it because of LESC and nrf53?

Reply:
Yes. It is added for nrf53